server/configuration.go

```diff
@@ -41,6 +41,8 @@ type configuration struct {
 
 	// ZoomWebhookSecret is the `Secret Token` taken from Zoom's webhook configuration page
 	ZoomWebhookSecret string
+
+	RestrictMeetingCreation bool
```

Comment:
Probably good to add a comment to this setting too

```suggestion
	// RestrictMeetingCreation allows the admin to by default restrict zoom meetings by default to only private channels. The admin can also edit each channel's behavior with the `/zoom channel-settings` command
	RestrictMeetingCreation bool
```